pkg/velerorunner/task.go

```diff
@@ -35,14 +38,29 @@ const (
 // Backup - A Backup created on the source cluster.
 // Restore - A Restore created on the destination cluster.
 type Task struct {
-	Log             logr.Logger
-	Client          k8sclient.Client
-	Owner           migapi.MigResource
-	PlanResources   *migapi.PlanResources
-	BackupResources []string
-	Phase           string
-	Backup          *velero.Backup
-	Restore         *velero.Restore
+	Log                   logr.Logger
+	Client                k8sclient.Client
+	Owner                 migapi.MigResource
+	PlanResources         *migapi.PlanResources
+	BackupResources       []string
+	Phase                 string
+	Backup                *velero.Backup
+	Restore               *velero.Restore
+	SrcRegistryResources  *MigrationRegistryResources
+	DestRegistryResources *MigrationRegistryResources
+}
+
+// The set of resources associated with a source or destination
+// Migration Registry.
+// CredSecret -       A secret containing the credentials for the registry storage
+// ImageStream -      The ImageStream resource pointing to the image used by the Registry DeploymentConfig
+// DeploymentConfig - The DeploymentConfig that runs the Migration Registry
+// Service          - The Service resource for the Migration Registry
+type MigrationRegistryResources struct {
```

Comment:
Suggest rename `RegistryResources`

Reply:
Suggest moving struct definition for this to `velerorunner/registry.go`

Reply:
Most of my registry-related names have Migration prepended to make it clear we're not talking about the internal registry. What about `MigRegistryResources`?

Reply:
I feel that since everything in our controller is aimed at performing migrations, it might be clear on its own. Do we expect to be interfacing with the internal OpenShift registry at any point?

Reply:
Also wonder if putting `Mig` in front of it makes it sound like that's one of our CRDs, which it isn't.

Reply:
The reason I hesitate on dropping the prefix completely is that we are still interacting with the internal registry in the plugin (which is where this registry gets used) -- basically the point of the migration registry is for the backup plugin to copy images from internal to migration registry, and for the restore plugin to copy images from migration registry to internal registry. Also, the annotation that we set on the velero CRs as a result of all of this is "openshift.io/migration-registry". What about MRegistry? It looks odd, but it won't be confused with "Mig-" names. I still like "Mig" or "MIgration" better than "M", but any of the above is better than omitting it. Also, we could go back to the longer name -- maybe that's OK if I make all the other name-shortening changes suggested.

Reply:
Oh I see. What if we went back to using the full name then?